src/openzaak/components/documenten/tests/utils.py

```diff
@@ -18,6 +19,10 @@ def get_operation_url(operation, **kwargs):
     )
 
 
+def get_documenten_spec() -> dict:
+    return get_spec("src/openzaak/components/documenten/openapi.yaml")
```

Comment:
this seems more like a generic test util for `openzaak.tests.utils`, taking a `component: str` argument.

Also, please use `pathlib.Path` to build filesystem paths instead of assuming unix.